ADAL/tests/app/src/ADTestAppSettings.m

```diff
@@ -75,7 +75,7 @@ + (void)initialize
        @"OneDrive"    : @{ @"authority" : @"https://login.microsoftonline.com/common",
                            @"resource" : @"https://api.office.com/discovery",
                            @"clientId" : @"af124e86-4e96-495a-b70a-90f90ab96707",
-                           @"redirectUri" : @"ms-onedrive://com.microsoft.skydrive",
+                           @"redirectUri" : @"ms-onedrive-auth://com.microsoft.skydrive",
```

Comment:
I think ms-onedrive should work already. Can you give a little bit more context on this change?

Reply:
@oldalton 
In the file MSALAdditionalTestAppSettings.h the profile for OneDive is 
@"OneDrive"    : @{ @"clientId" : @"af124e86-4e96-495a-b70a-90f90ab96707",
                    @"redirectUri" : @"**ms-onedrive-auth:**//com.microsoft.skydrive",
} 
So It's a tiny update to make ADAL Test App consistent with others.

Reply:
In a second thought, because using ms-onedrive is still working so this change is unnecessary. So it's might not worth merging into dev. Sorry about the redundancy.

Reply:
I'm fine with both :)